docs/get-started/using-standalone.md

```diff
@@ -20,20 +20,22 @@ import {Deck} from '@deck.gl/core';
 import {ScatterplotLayer} from '@deck.gl/layers';
 
 const INITIAL_VIEW_STATE = {
-  latitude: 37.78,
+  latitude: 37.8,
   longitude: -122.45,
-  zoom: 3
+  zoom: 15
 };
 
 const deckgl = new Deck({
-  canvas: 'my-deck-canvas',
+  canvas: 'my-deck-canvas', // This is the canvas id in your html (<canvas id="my-deck-canvas">)
```

Comment:
We can just remove this line. The default parent will be document.body

Reply:
I've removed it, but I feel like most users may be more comfortable with mounting to a dom element, rather than it being automatically created